src/compiler/compile/render_dom/wrappers/Element/index.ts

```diff
@@ -906,6 +907,29 @@ export default class ElementWrapper extends Wrapper {
 		});
 	}
 
+	add_styles(block: Block) {
+		this.node.styles.forEach((style_directive) => {
+			const { name, expression, text } = style_directive;
+
+			let snippet;
+			if (text) {
+				snippet = `"${text}"`;
+			} else if (expression) {
+				snippet = expression.manipulate(block);
+			} else {
+				snippet = name;
+			}
```

Comment:
Covers the three variants:
`<p style:color="red" />`
`<p style:color={myColor} />`
`<p style:color />`